slim.Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Build wheels
+FROM python:3.9-slim as build
```

Comment:
Should we use Python 3.10 in preference to 3.9?  Or, what should be our criterion for choosing a base python?

Reply:
With that said, we can easily add a build argument and publish 3.8, 3.9, and 3.10 images

Reply:
Further thoughts - we should use the same Python version that the Girder4 cookie-cutter uses: https://github.com/girder/cookiecutter-girder-4/blob/d1912b887133ae2407277f772f6329c082fafb73/%7B%7B%20cookiecutter.project_slug%20%7D%7D/dev/django.Dockerfile#L1